README.md

```diff
@@ -264,6 +286,20 @@ $ cd build
 $ cmake -DVCPKG_TARGET_TRIPLET=x64-windows -DCMAKE_TOOLCHAIN_FILE='/vcpkg/scripts/buildsystems/vcpkg.cmake' -DCMAKE_INSTALL_PREFIX=install -DTRITON_ENABLE_CC_GRPC=ON -DTRITON_ENABLE_PYTHON_GRPC=ON -DTRITON_ENABLE_GPU=OFF -DTRITON_ENABLE_EXAMPLES=ON -DTRITON_ENABLE_TESTS=ON ..
 ```
 
+If you are building on a release branch (or on a development branch
+that is based off of a release branch), then you must also use
+additional cmake arguments to point to that release branch for repos
+that the client build depends on. For example, if you are building the
+r21.10 client branch then you need to use the following additional
+cmake flags:
+
+```
+-DTRITON_COMMON_REPO_TAG=r21.10
+-DTRITON_THIRD_PARTY_REPO_TAG=r21.10
+-DTRITON_CORE_REPO_TAG=r21.10
+-DTRITON_BACKEND_REPO_TAG=r21.10
+```
+
```

Comment:
Would it be better to add these to another section eg `#### Additional Build Args` before or after the windows/non-windows build instructions since they are the same?

Reply:
I'd rather keep them close to the existing cmake as otherwise I think they may not be noticed.